afew/commands.py

```diff
@@ -74,6 +74,10 @@
     '-d', '--dry-run', default=False, action='store_true',
     help="don't change the db [default: %(default)s]"
 )
+options_group.add_argument(
+    '-N', '--no-new', default=False, action='store_true',
+    help="do not run notmuch new after moving mails, you will have to run it manualy"
```

Comment:
I'd suggest something more explaining like this:
> do not run `notmuch new` after moving mails, to avoid looping if afew is configured as a notmuch hook

I'd strip the `-N`, as it doesn't really speak for itself.